exporter/exporter.go

```diff
@@ -267,6 +268,21 @@ func NewRedisExporter(redisURI string, opts Options) (*Exporter, error) {
 			"server_threads":        "server_threads_total",
 			"long_lock_waits":       "long_lock_waits_total",
 			"current_client_thread": "current_client_thread",
+
+			// Redis Modules metrics
+			// RediSearch module
+			"search_number_of_indexes":   "search_number_of_indexes",
+			"search_used_memory_indexes": "search_used_memory_indexes_bytes",
+			"search_total_indexing_time": "search_total_indexing_time_ms",
```

Comment:
`search_total_indexing_time` is apparently a counter, will prepare PR with renaming it to `search_indexing_time_ms_total`